components/AppBar/README.md

```diff
@@ -76,9 +76,9 @@ The Material Design top app bar displays information and actions relating to the
   - [Enabling top layout guide adjustment](#enabling-top-layout-guide-adjustment)
   - [Enabling inferred top safe area insets](#enabling-inferred-top-safe-area-insets)
 - [Extensions](#extensions)
-  - [Color Theming](#color-theming)
-  - [Typography Theming](#typography-theming)
-  - [Theming Extensions (Beta)](#theming-extensions-(beta))
+  - [Color Theming (To Be Deprecated)](#color-theming-(to-be-deprecated))
+  - [Typography Theming (To Be Deprecated)](#typography-theming-(to-be-deprecated))
```

Comment:
Please pick a term and be consistent; another PR used “to be deleted” instead.

Reply:
Done

Reply:
@jverkoey Aren't the two categories for distinctly different parts of the deprecation process?

*  *To be deprecated* - Early stage, not yet deprecated.
*  *To be deleted* - Very late stage, already deprecated and soon removed entirely.

Reply:
Yep; as discussed offline, let's create a new table of contents section for "Unsupported" APIs that are on their way out.